examples/calculator/run_calculator.py

```diff
@@ -9,7 +9,7 @@
 @session.startup(
     url='http://calc',
     image_hash='25f09e17c34433f979331edf4f3b47b2ca330ba2f8acbfe2e3dbd9c3',
-    service_cnt=3,
+    init_size=3,
```

Comment:
`init_size` name doesn't suggest size *of what* is specified. Is it the size of the session? But what does *that* mean? Maybe `init_num_providers`, or `init_provider_cnt`, or `init_cluster_size`? The same applies to `session.set_size()`

Reply:
55838e3